home/modules/contribute/pages/includes.adoc

```diff
@@ -105,6 +105,21 @@ For example, if you tag code in a YAML file, place a hash (`+#+`) before the tag
     - public/
 ----
 
+== Couchbase Documentation Partials
+
+=== Developer Preview Partial
+
+To call out a section of a page as Developer Preview, use the `developer-preview` partial.
+Otherwise if the whole page is Developer Preview content, you can use the xref:attributes-and-roles.adoc#custom-page-attributes[page-status] attribute.
+
+----
+\include::home:ROOT:partial$developer-preview.adoc[]
```

Comment:
Should there be a note here about the more typical PATH found in most CB components (ie, not /home/)?

Reply:
In theory that path will be the same regardless of where the partial is included (but haven't tried it yet). The idea is to have a single partial which can be included throughout the site's components. I've just added the staging link from the PR checks on the PR's 1st comment so you can now see the output.

Reply:
Okay, we'll see how it goes :+1:

Reply:
Yes 🙂